### PR TITLE
bcolferzd/release flows

### PR DESCRIPTION
**Pull request by bcolferzd**
- Purpose of the future:
  Release flow provides a structured release sequence 
- Screenshot of the UI
  screen shot 2016-10-25 at 13 15 49
- Unit tests:
  application_controller_test.rb
  release_controller_test.rb

/cc @zendesk/samson
### Tasks
- [ ] :+1: from team
### References
- Jira link: https://zendesk.atlassian.net/browse/SAMSON-297
### Risks
- Level: Low/Med/High

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -113,3 +113,7 @@ GITHUB_TOKEN=
 # JENKINS_USERNAME= # user id
 # JENKINS_API_KEY= # API Token from user / Configure page
 # JENKINS_NOTIFY_COMMITTERS=1 # optional: send committers to jenkins as 'emails' params (filtered by GOOGLE_DOMAIN)
+
+## Release flow:  Deploy production in a defined order
```
Comment (grosser) [inline .env.example]: "deploy stages"

File: app/views/releases/_release.html.erb [hunk 2]
```diff
@@ -21,6 +21,9 @@
   </td>
   <td>
     <% if deployer_for_project? %>
+      <% if ENV['RELEASE_FLOW'] %>
+        <%= link_to "Release Flow", flow_project_release_path(@project, release) %>
```
Comment (grosser) [inline app/views/releases/_release.html.erb]: this still needs some css love ...

**Comment by grosser** [comment]
👍

**Comment by bcolferzd** [comment]
Going with another direction ... adding this functionality as a plugin

**Pull request closed without merging**